frontend/src/components/form/base/BasePicker.vue

```diff
@@ -3,11 +3,11 @@ Displays a field as a picker (can be used with v-model)
 -->
 
 <template>
-  <div class="e-form-container">
+  <div ref="picker" class="e-form-container">
     <v-menu
-      ref="menu"
       v-model="showPicker"
       :disabled="disabled || readonly"
+      :close-on-click="false"
```

Comment:
Why is this needed? The effect is now, that pickers are not closing when I click on another picker activator.

![image](https://user-images.githubusercontent.com/449555/205658134-9faf8d4f-dec6-4cc6-a72a-3a767165597b.png)

Reply:
It is needed because the timepicker requires multiple clicks to select a time (first click an hour, then click a minute). TBH I have no idea how it worked before. But `close-on-click="true"` should mean that the picker is closed on every click if everything is set up correctly.

Reply:
It actually shouldn't because you already have `:close-on-content-click="false"` on the next line. At least in chrome it's working for me, even if I remove `:close-on-click="false"`